spring-cloud-stream/pom.xml

```diff
@@ -35,6 +35,10 @@
 			<artifactId>spring-integration-core</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-jmx</artifactId>
```

Comment:
Yes, but the metrics endpoint is conditional on JMX in Boot so for now we need to have it.